server/db/clean-database-backup.sql

```diff
@@ -0,0 +1,27 @@
+-- sql for purging pools and its' related entities.
+-- helpful for database backups where ExpiredPoolsJob has not run for a long time.
+SET FOREIGN_KEY_CHECKS=0;
+
+DELETE FROM cp_pool_attribute WHERE pool_id = (SELECT id FROM cp_pool WHERE enddate < now() AND id = cp_pool_attribute.pool_id);
```

Comment:
The subquery here was acceptable in our immediate case, but in the general case, doesn't account for situations where an entitlement extends the end date of a pool.